app/move/fields/index.js

```diff
@@ -85,12 +89,12 @@ const createFields = {
   police_national_computer: policeNationalComputer,
   pregnant: assessmentAnswer(),
   prison_recall_comments: {
-    ...cloneDeep(additionalInformation),
+    ...cloneDeep(additionalInformationBase),
```

Comment:
So I think in other instances [I've imported the shared attributes into the field itself](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/d7dede31c20fb203a087b35b841c6bf732d75dd7/app/move/fields/date-to.js#L3) to try and remove the logic from this index loader.

Reply:
That's orthogonal to this commit!  Raise an issue!

Reply:
Done. Obviously we have all the `assessment()` and other thangs there awaiting a clean out